source/dubregistry/registry.d

```diff
@@ -570,6 +655,15 @@ struct PackageInfo {
 	Json info; /// JSON package information, as reported to the client
 }
 
+/// flags to customize getPackageInfo* methods
+enum PackageInfoFlags
+{
+	none,
+	include_dependencies = 1 << 0, /// include package info of dependencies
```

Comment:
Naming convention: `includeDependencies`/`includeErrors`

Reply:
Fixed this (only for this flag, not for the web interface)